sdk/identity/azure-identity/azure/identity/_internal/aad_client_base.py

```diff
@@ -63,6 +66,19 @@ def get_cached_refresh_tokens(self, scopes):
         """Assumes all cached refresh tokens belong to the same user"""
         return self._cache.find(TokenCache.CredentialType.REFRESH_TOKEN, target=list(scopes))
 
+    def is_refresh(self, token):
```

Comment:
I use is_refresh because we have is_retry in core to check if we need retry or not.

Reply:
Ah, I see. I'd rename that one too, for similar reasons 😆. I'd call it `should_retry` because it answers the question "given this response, should I retry the request?" `is_retry(retry_settings, response)` suggests checking whether the response has the property or nature of being a retry, which I suppose would mean being a response to a retried request?

Reply:
Agree should is better. But we cannot rename that one because that is a public method . :( 

And that's the reason I chose to align with that one.

Reply:
Why is that alignment important? `AadClient` is an internal class in a different library, and semantically unrelated to `RetryPolicy`.